composer.json

```
@@ -1,36 +1,44 @@
 {
-	"name": "dnj/laravel-invoice",
-	"license": "MIT",
-	"autoload": {
+    "name": "dnj/laravel-ticketing",
+    "description": "Package used for the ticketing system.",
+    "type": "library",
+    "require": {
+        "php": ">=8.1",
```

```suggestion
        "php": "^8.1",
```
This make php >= 8.1 < 9.0